packages/array/index.d.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Create a numeric range
+ *
+ * @param {Number} from
+ * @param {Number} to
+ * @return {Array}
+ *
+ * @example
+ * Array.range(-2, 2) // => [-2, -1, 0, 1, 2]
+ * Array.range(2, -2) // => [2, 1, 0, -1, -2]
+ */
+export declare function range(from: number, to: number): any[];
+/**
+ *
+ * Rotates a list a number of times. It"s completly agnostic about the
+ * contents of the list.
+ *
+ * @param {Integer} times - the number of rotations
+ * @param {Array} array
+ * @return {Array} the rotated array
+ * @example
+ * Array.rotate(1, [1, 2, 3]) // => [2, 3, 1]
+ */
+export declare function rotate(times: number, arr: any[]): any;
```

Comment:
It always returns an array:
```suggestion
export declare function rotate(times: number, arr: any[]): any[];
```